pkg/cli/images_sign.go

```diff
@@ -0,0 +1,165 @@
+package cli
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"strings"
+
+	cli "github.com/acorn-io/runtime/pkg/cli/builder"
+	"github.com/acorn-io/runtime/pkg/client"
+	acornsign "github.com/acorn-io/runtime/pkg/cosign"
+	"github.com/acorn-io/runtime/pkg/images"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/pterm/pterm"
+	"github.com/sigstore/cosign/v2/cmd/cosign/cli/generate"
+	"github.com/sigstore/cosign/v2/pkg/cosign"
+	"github.com/sigstore/cosign/v2/pkg/cosign/remote"
+	"github.com/sigstore/cosign/v2/pkg/oci/mutate"
+	ociremote "github.com/sigstore/cosign/v2/pkg/oci/remote"
+	"github.com/sigstore/cosign/v2/pkg/oci/static"
+	"github.com/sigstore/cosign/v2/pkg/signature"
+	sigsig "github.com/sigstore/sigstore/pkg/signature"
+	"github.com/spf13/cobra"
+)
+
+func NewImageSign(c CommandContext) *cobra.Command {
+	cmd := cli.Command(&ImageSign{client: c.ClientFactory}, cobra.Command{
+		Use:               "sign IMAGE_NAME [flags]",
+		Example:           `acorn image sign my-image --key ./my-key`,
```

Comment:
Is it possible to pass a GitHub key here? If so, can we add another example to illustrate this?

Reply:
gh://, ac://, etc. are only supported for verification, where we need public keys